front/lib/workspace_usage.ts

```diff
@@ -58,6 +58,7 @@ type builderUsageQueryResult = {
   userLastName: string;
   agentsEditionsCount: number;
   distinctAgentsEditionsCount: number;
+  lastEditAt: string;
```

Comment:
nit: types usually start with a capitalized letter (for one of your previous pr)